engines/dfc_provider/app/services/fdc_request.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require "private_address_check"
+require "private_address_check/tcpsocket_ext"
+
+# Request a JSON document from the FDC API with authentication.
+#
+# Currently, the API doesn't quite comply with the DFC standard and we need
+# to authenticate a little bit differently.
```

Comment:
I though DFC did not specify authentication, but I was wrong : https://datafoodconsortium.gitbook.io/dfc-standard-documentation/technical-specifications/authentication-strategy